evaluate_summaries.py

```diff
@@ -31,19 +30,13 @@
     if args.data_subset == "full":
         test_set_ids = set(xsum_test.keys())
     elif args.data_subset == "full-extrinsic":
-        test_set = load_extrinsic_test_set(
-            xsum_test, baseline_metadata, gold_metadata, 10000
+        test_set_ids = set(
+            load_shuffled_test_split(xsum_test, "test-extrinsic", 10000).keys()
         )
-        test_set_ids = {k for (k, v) in test_set}
     else:
-        test_set = (
-            load_extrinsic_test_set(
-                xsum_test, baseline_metadata, gold_metadata, args.test_size
-            )
-            if args.data_subset == "test-extrinsic"
-            else load_xent_test_set(xsum_test, gold_metadata, args.test_size)
+        test_set_ids = set(
```

Comment:
I verified that this change works as intended by running evaluate_summaries with test-size 100 (it yields the same results).